CMakeLists.txt

```diff
@@ -16,7 +16,16 @@ set(VM_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src/vm)
 
 if(CMAKE_SYSTEM_NAME STREQUAL Linux)
     set(CLR_CMAKE_PLATFORM_UNIX 1)
-    set(CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64 1)
+    if(CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64)
+        set(CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64 1)
+    elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l)
+        set(CLR_CMAKE_PLATFORM_UNIX_TARGET_ARM 1)
+        # Because we don't use CMAKE_C_COMPILER/CMAKE_CXX_COMPILER to use clang
+        # we have to set the triple by adding a compiler argument
+        add_compile_options(-target armv7-linux-gnueabihf)
+        add_compile_options(-mthumb)
+        add_compile_options(-mfpu=vfpv3)
```

Comment:
How would you suggest this is done, only examples I can see are to get the output of /proc/cpuinfo in CMake and see if features are available. Also, whilst VFPv3 may not be guaranteed, at least in Cortex cores, it's included in all but the A5 and A9 where it is supposedly optional, but I haven't seen an example lacking it.